### PR TITLE
add presentations part in doc website

### PR DESCRIPTION
**Pull request by helenlly**
add presentations part in doc website
add 2018 presentations link

### CODE REVIEW

File: docs/docs/presentations.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+**Talks:**
+
+* [Strata 2018 San Jose:](https://conferences.oreilly.com/strata/strata-ca-2018/public/schedule/detail/64023)Automatic 3D MRI knee damage classification with 3D CNN using BigDL on Spark[[Slides]](https://cdn.oreillystatic.com/en/assets/1/event/269/Automatic%203D%20MRI%20knee%20damage%20classification%20with%203D%20CNN%20using%20BigDL%20on%20Spark%20Presentation.pdf)
```
Comment (hkvision) [inline docs/docs/presentations.md]: Add a space before `Automatic`? also one extra space for other items.

File: docs/docs/presentations.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+* [Spark+AI Summit 2018 SF:](https://databricks.com/session/analytics-zoo-building-analytics-and-ai-pipeline-for-apache-spark-and-bigdl)Analytics Zoo: Building Analytics and AI Pipeline for Apache Spark and BigDL[[Video]](https://jason-dai.github.io/cvpr2018/slides/BigData_DL_Jason-CVPR.pdf)
+
+* [Spark+AI Summit 2018 SF:](https://databricks.com/session/building-deep-reinforcement-learning-applications-on-apache-spark-using-bigdl)Building Deep Reinforcement Learning Applications on Apache Spark with Analytics Zoo using BigDL[[Slides]](https://github.com/analytics-zoo/analytics-zoo.github.io/blob/master/presentations/Building%20Deep%20Reinforcement%20Learning%20Applications%20on%20Apache%20Spark%20with%20Analytics%20Zoo%20using%20BigDL.pdf)
+
+* [Spark+AI Summit 2018 SF:](https://databricks.com/session/using-crowdsourced-images-to-create-image-recognition-models-with-bigdl)Using Crowdsourced Images to Create Image Recognition Models with Analytics Zoo using BigDL[[Slides]](https://github.com/analytics-zoo/analytics-zoo.github.io/blob/master/presentations/Using%20Crowdsourced%20Images%20to%20Create%20Image%20Recognition%20Models%20with%20Analytics%20Zoo%20using%20BigDL.pdf)
```
Comment (hkvision) [inline docs/docs/presentations.md]: group talks in the same conference together?
```
Spark+AI Summit 2018 SF:
     - xxx
     - xxx
     - xxx
```

File: docs/docs/presentations.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
+**Talks:**
+
+* [Accelerating analytics and AI from the edge to the cloud](https://conferences.oreilly.com/strata/strata-ca-2018/public/schedule/detail/66393)  *[[Slides Download]](https://cdn.oreillystatic.com/en/assets/1/event/269/Accelerating%20analytics%20and%20AI%20from%20the%20edge%20to%20the%20cloud%20_sponsored%20by%20Intel_%20Presentation.pdf)*<br />(Strata Data conference, March 2018, San Jose)
```
Comment (jason-dai) [inline docs/docs/presentations.md]: Don't need this one

File: docs/docs/presentations.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+
+* [Automatic 3D MRI knee damage classification with 3D CNN using BigDL on Spark](https://conferences.oreilly.com/strata/strata-ca-2018/public/schedule/detail/64023)  *[[Slides Download]](https://cdn.oreillystatic.com/en/assets/1/event/269/Automatic%203D%20MRI%20knee%20damage%20classification%20with%203D%20CNN%20using%20BigDL%20on%20Spark%20Presentation.pdf)*<br /> (Strata Data conference, March 2018, San Jose)
```
Comment (jason-dai) [inline docs/docs/presentations.md]: * **Automatic 3D MRI knee damage classification with 3D CNN using BigDL on Spark**, [Strata Data Conference](https://conferences.oreilly.com/strata/strata-ca-2018/public/schedule/detail/64023), March 2018, San Jose ([slides](https://cdn.oreillystatic.com/en/assets/1/event/269/Automatic%203D%20MRI%20knee%20damage%20classification%20with%203D%20CNN%20using%20BigDL%20on%20Spark%20Presentation.pdf))

File: docs/docs/presentations.md [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+
+* [Running distributed Keras based on Apache Spark and BigDL](https://ai.oreilly.com.cn/ai-cn-2018/public/schedule/detail/64845)  *[[Slides Download]](https://github.com/analytics-zoo/analytics-zoo.github.io/blob/master/presentations/Analytics%20Zoo%20tutorial.pdf)*<br />(AI Conference, April 2018, Beijing)
+
+* [Build deep learning-powered big data solutions with BigDL](https://conferences.oreilly.com/artificial-intelligence/ai-ny-2018/public/schedule/detail/68590)  *[[Slides Download]](https://cdn.oreillystatic.com/en/assets/1/event/280/Build%20deep%20learning-powered%20big%20data%20solutions%20with%20BigDL%20Presentation.pdf)*<br /> (AI conference, May 2018, New York)
```
Comment (jason-dai) [inline docs/docs/presentations.md]: Don't need this one

File: docs/docs/presentations.md [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+
+* [Classifying images in Spark](https://conferences.oreilly.com/artificial-intelligence/ai-ny-2018/public/schedule/detail/64939)  *[[Slides Download]](https://cdn.oreillystatic.com/en/assets/1/event/280/Classifying%20images%20in%20Spark%20Presentation.pdf)*<br /> (AI conference, May 2018, New York)
```
Comment (jason-dai) [inline docs/docs/presentations.md]: Classifying images on Spark in World Bank

File: docs/docs/presentations.md [hunk 7]
```diff
@@ -0,0 +1,33 @@
...
+
+* [Accelerating deep learning on Apache Spark using BigDL with coarse-grained scheduling](https://conferences.oreilly.com/strata/strata-ca-2018/public/schedule/detail/63960)  *[[Slides Download]](https://cdn.oreillystatic.com/en/assets/1/event/269/Accelerating%20deep%20learning%20on%20Apache%20Spark%20using%20BigDL%20with%20coarse-grained%20scheduling%20Presentation.pptx)*<br /> (Strata Data conference, March 2018, San Jose)
+
+* [Improving user-merchant propensity modeling using neural collaborative filtering and wide and deep models on Spark BigDL at scale](https://conferences.oreilly.com/strata/strata-ca-2018/public/schedule/detail/63897)  *[[Slides Download]](https://cdn.oreillystatic.com/en/assets/1/event/269/Improving%20user-merchant%20propensity%20modeling%20using%20neural%20collaborative%20filtering%20and%20wide%20and%20deep%20models%20on%20Spark%20BigDL%20at%20scale%20Presentation.pdf)*<br /> (Strata Data conference, March 2018, San Jose)
```
Comment (jason-dai) [inline docs/docs/presentations.md]: Improving user-merchant propensity modeling using neural collaborative filtering and wide and deep models on Spark BigDL in Mastercard

File: docs/docs/presentations.md [hunk 8]
```diff
@@ -0,0 +1,33 @@
...
+
+* [Building Deep Reinforcement Learning Applications on Apache Spark with Analytics Zoo using BigDL](https://databricks.com/session/building-deep-reinforcement-learning-applications-on-apache-spark-using-bigdl)  *[[Slides Download]](https://github.com/analytics-zoo/analytics-zoo.github.io/blob/master/presentations/Building%20Deep%20Reinforcement%20Learning%20Applications%20on%20Apache%20Spark%20with%20Analytics%20Zoo%20using%20BigDL.pdf)*<br /> (Spark+AI Summit, June 2018, San Francisco)
+
+* [Using Crowdsourced Images to Create Image Recognition Models with Analytics Zoo using BigDL](https://databricks.com/session/using-crowdsourced-images-to-create-image-recognition-models-with-bigdl)  *[[Slides Download]](https://github.com/analytics-zoo/analytics-zoo.github.io/blob/master/presentations/Using%20Crowdsourced%20Images%20to%20Create%20Image%20Recognition%20Models%20with%20Analytics%20Zoo%20using%20BigDL.pdf)*<br /> (Spark+AI Summit, June 2018, San Francisco)
```
Comment (jason-dai) [inline docs/docs/presentations.md]: Using Crowdsourced Images to Create Image Recognition Models with Analytics Zoo in World Bank

**Comment by jason-dai** [comment]
Please fix the format

File: docs/docs/presentations.md [hunk 9]
```diff
@@ -0,0 +1,27 @@
+**Talks:**
+
+* Automatic 3D MRI knee damage classification with 3D CNN using BigDL on Spark in UCSF, [Strata Data conference](https://conferences.oreilly.com/strata/strata-ca-2018/public/schedule/detail/64023), March 2018, San Jose [(slides)](https://cdn.oreillystatic.com/en/assets/1/event/269/Automatic%203D%20MRI%20knee%20damage%20classification%20with%203D%20CNN%20using%20BigDL%20on%20Spark%20Presentation.pdf)
+
+* Accelerating deep learning on Apache Spark using BigDL with coarse-grained scheduling, <br />[Strata Data conference](https://conferences.oreilly.com/strata/strata-ca-2018/public/schedule/detail/63960), March 2018, San Jose [(slides)](https://cdn.oreillystatic.com/en/assets/1/event/269/Accelerating%20deep%20learning%20on%20Apache%20Spark%20using%20BigDL%20with%20coarse-grained%20scheduling%20Presentation.pptx)
```
Comment (hkvision) [inline docs/docs/presentations.md]: Keep everything organized? For example, whether all the titles have a link and use [slides] or [Slides Download]

File: docs/docs/presentations.md [hunk 10]
```diff
@@ -0,0 +1,27 @@
...
+
+* [A deep learning approach for precipitation nowcasting with RNN using Analytics Zoo on BigDL](https://conferences.oreilly.com/strata/strata-ny-2018/public/schedule/detail/69413)  *[[Slides Download]](https://github.com/analytics-zoo/analytics-zoo.github.io/blob/master/presentations/A%20deep%20learning%20approach%20for%20precipitation%20nowcasting%20with%20RNN%20using%20Analytics%20Zoo%20on%20BigDL.pdf)*<br /> (Strata Data conference, Sep. 2018, New York)
+
+* [Use Intel Analytics Zoo to build an intelligent QA Bot for Microsoft Azure](https://www.meetup.com/Shanghai-Apache-Spark-AI-Meetup/)   *[[Slides Download]](https://github.com/analytics-zoo/analytics-zoo.github.io/blob/master/presentations/Use%20Intel%20Analytics%20Zoo%20to%20build%20an%20intelligent%20QA%20Bot%20for%20Microsoft%20Azure.pdf)* <br /> (Shanghai Apache Spark+AI meetup, Nov 2018, Shanghai)
```
Comment (hkvision) [inline docs/docs/presentations.md]: it is strange that you link the title to the conference/meetup info.....

**Comment by helenlly** [comment]
any comments? @jason-dai @hkvision

File: docs/docs/presentations.md [hunk 11]
```diff
@@ -0,0 +1,27 @@
+**Talks:**
+
+* Automatic 3D MRI knee damage classification with 3D CNN using BigDL on Spark in UCSF, [Strata Data conference](https://conferences.oreilly.com/strata/strata-ca-2018/public/schedule/detail/64023), March 2018, San Jose([slides](https://cdn.oreillystatic.com/en/assets/1/event/269/Automatic%203D%20MRI%20knee%20damage%20classification%20with%203D%20CNN%20using%20BigDL%20on%20Spark%20Presentation.pdf))
```
Comment (jason-dai) [inline docs/docs/presentations.md]: Automatic 3D MRI knee damage classification with 3D CNN using BigDL on Spark in UCSF, [Strata Data conference](https://conferences.oreilly.com/strata/strata-ca-2018/public/schedule/detail/64023), March 2018, San Jose ([slides](https://cdn.oreillystatic.com/en/assets/1/event/269/Automatic%203D%20MRI%20knee%20damage%20classification%20with%203D%20CNN%20using%20BigDL%20on%20Spark%20Presentation.pdf))

Comment (hkvision) [reply docs/docs/presentations.md]: Add a space before (slides)

File: docs/docs/presentations.md [hunk 12]
```diff
@@ -0,0 +1,27 @@
...
+
+* Classifying images on Spark in World Bank, [AI conference](https://conferences.oreilly.com/artificial-intelligence/ai-ny-2018/public/schedule/detail/64939), May 2018, New York([slides](https://cdn.oreillystatic.com/en/assets/1/event/280/Classifying%20images%20in%20Spark%20Presentation.pdf))
+
+* Using Siamese CNNs for removing duplicate entries from real estate listing databases, [Strata Data conference](https://conferences.oreilly.com/strata/strata-eu-2018/public/schedule/detail/65518), May 2018, London([slides](https://cdn.oreillystatic.com/en/assets/1/event/267/Using%20Siamese%20CNNs%20for%20removing%20duplicate%20entries%20from%20real%20estate%20listing%20databases%20Presentation.pdf))
+
+* Analytics Zoo: Building Analytics and AI Pipeline for Apache Spark and BigDL, [Spark+AI Summit](https://databricks.com/session/analytics-zoo-building-analytics-and-ai-pipeline-for-apache-spark-and-bigdl), June 2018, San Francisco
+
+* Using BigDL on Apache Spark to Improve the MLS Real Estate Search Experience at Scale, [Spark+AI Summit](https://databricks.com/session/using-bigdl-on-apache-spark-to-improve-the-mls-real-estate-search-experience-at-scale), June 2018, San Francisco
```
Comment (hkvision) [inline docs/docs/presentations.md]: no slides available for these two?

File: docs/docs/presentations.md [hunk 13]
```diff
@@ -0,0 +1,27 @@
...
+
+* Job recommendations leveraging deep learning using Analytics Zoo on Apache Spark and BigDL, [Strata Data conference](https://conferences.oreilly.com/strata/strata-ny-2018/public/schedule/detail/69113), Sep. 2018, New York([slides](https://cdn.oreillystatic.com/en/assets/1/event/278/Job%20recommendations%20leveraging%20deep%20learning%20using%20Analytics%20Zoo%20on%20Apache%20Spark%20and%20BigDL%20Presentation.pdf))
+
+* A deep learning approach for precipitation nowcasting with RNN using Analytics Zoo on BigDL, [Strata Data conference](https://conferences.oreilly.com/strata/strata-ny-2018/public/schedule/detail/69413), Sep. 2018, New York([slides](https://github.com/analytics-zoo/analytics-zoo.github.io/blob/master/presentations/A%20deep%20learning%20approach%20for%20precipitation%20nowcasting%20with%20RNN%20using%20Analytics%20Zoo%20on%20BigDL.pdf))
```
Comment (hkvision) [inline docs/docs/presentations.md]: Sep. -> Sep ?

File: docs/docs/presentations.md [hunk 14]
```diff
@@ -0,0 +1,27 @@
...
+
+* Use Intel Analytics Zoo to build an intelligent QA Bot for Microsoft Azure, [Shanghai Apache Spark+AI meetup](https://www.meetup.com/Shanghai-Apache-Spark-AI-Meetup/), Nov 2018, Shanghai([slides](https://github.com/analytics-zoo/analytics-zoo.github.io/blob/master/presentations/Use%20Intel%20Analytics%20Zoo%20to%20build%20an%20intelligent%20QA%20Bot%20for%20Microsoft%20Azure.pdf))
```
Comment (hkvision) [inline docs/docs/presentations.md]: change link to https://www.meetup.com/Shanghai-Apache-Spark-AI-Meetup/events/255788956/ ?

File: docs/docs/presentations.md (hunk 12 above)
Comment (helenlly) [reply docs/docs/presentations.md]: no slides

File: docs/docs/presentations.md (hunk 11 above)
Comment (helenlly) [reply docs/docs/presentations.md]: done

File: docs/docs/presentations.md (hunk 14 above)
Comment (helenlly) [reply docs/docs/presentations.md]: done

File: docs/docs/presentations.md (hunk 13 above)
Comment (helenlly) [reply docs/docs/presentations.md]: done

File: docs/docs/presentations.md [hunk 15]
```diff
@@ -0,0 +1,29 @@
...
+
+* Use Intel Analytics Zoo to build an intelligent QA Bot for Microsoft Azure, [Shanghai Apache Spark+AI meetup](https://www.meetup.com/Shanghai-Apache-Spark-AI-Meetup/events/255788956/), Nov 2018, Shanghai ([slides](https://github.com/analytics-zoo/analytics-zoo.github.io/blob/master/presentations/Use%20Intel%20Analytics%20Zoo%20to%20build%20an%20intelligent%20QA%20Bot%20for%20Microsoft%20Azure.pdf))
+
+* Analytics Zoo: Unifying Big Data Analytics and AI for Apache Spark, [Shanghai Apache Spark+AI meetup](https://www.meetup.com/Shanghai-Apache-Spark-AI-Meetup/events/255788956/), Nov 2018, Shanghai ([slides](https://github.com/analytics-zoo/analytics-zoo.github.io/blob/master/presentations/Analytics%20Zoo-Unifying%20Big%20Data%20Analytics%20and%20AI%20for%20Apache%20Spark.pdf))
```
Comment (hkvision) [inline docs/docs/presentations.md]: put this before the previous one?

Comment (helenlly) [reply docs/docs/presentations.md]: done

**Comment by hkvision** [comment]
LGTM

**Pull request merged**